### PR TITLE
update comments

### PR DESCRIPTION
**Pull request by grandoc**

### CODE REVIEW

File: htdocs/core/modules/commande/doc/pdf_eratosthene.modules.php [hunk 1]
```diff
@@ -68,26 +68,57 @@ class pdf_eratosthene extends ModelePDFCommandes
     public $type;
 
 	/**
-     * @var array Minimum version of PHP required by module.
-	 * e.g.: PHP ≥ 5.3 = array(5, 3)
+     * @var array() Minimum version of PHP required by module.
```
Comment (frederic34) [inline htdocs/core/modules/commande/doc/pdf_eratosthene.modules.php]: @var array is right

Comment (frederic34) [reply htdocs/core/modules/commande/doc/pdf_eratosthene.modules.php]: http://docs.phpdoc.org/references/phpdoc/types.html#keyword

File: htdocs/core/modules/facture/doc/pdf_sponge.modules.php [hunk 2]
```diff
@@ -69,26 +69,57 @@ class pdf_sponge extends ModelePDFFactures
...
+     * @var array() Minimum version of PHP required by module.
+	 * e.g.: PHP ≥ 5.4 = array(5, 4)
      */
-	public $phpmin = array(5, 2);
+	public $phpmin = array(5, 4);
 
 	/**
      * Dolibarr version of the loaded document
-     * @var string
+     * @public string
```
Comment (frederic34) [inline htdocs/core/modules/facture/doc/pdf_sponge.modules.php]: tag `@public` does'nt exists, keep `@var`
http://docs.phpdoc.org/references/phpdoc/tags/var.html

**Comment by grandoc** [comment]
Done...

**Pull request merged**